src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WebSearch/Helpers/HistoryItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text.Json;
+
+namespace Microsoft.CmdPal.Ext.WebSearch.Helpers;
+
+public class HistoryItem(string searchString, DateTime timestamp)
+{
+    public string SearchString { get; private set; } = searchString;
+
+    public DateTime Timestamp { get; private set; } = timestamp;
+
+    public string ToJson() => JsonSerializer.Serialize(this);
```

Comment:
> `JsonSerializer.Serialize(this);`

that just.... works? what the hell is this witchcraft language

Reply:
It appears to work. Did you play around at all with history items yourself? They do just seem to work perfectly for me.